packages/components/src/ListView/Items/__snapshots__/items.test.js.snap

```diff
@@ -280,28 +280,65 @@ exports[`Items should render 1`] = `
                       }
                       measure={[Function]}
                     >
-                      <div>
-                        <div
-                          className="checkbox-container"
+                      <I18n
```

Comment:
Like @andreasmattes said in an other way, I don't think the actual snapshots should have been updated (not so much though) if keeping the underlying component for most of the cases, and just adding some tests for translation.

Reply:
It should. This is on `Items` that import `Item`. But `Item` default export has changed, it's not the component but the i18n wrapped component. That's why we have this difference